src/scripts/Providers/OSUI/SharedProviderResources/Flatpickr/scss/_flatpickr.scss

```diff
@@ -320,11 +319,22 @@ span.flatpickr-weekday {
 		}
 	}
 
-	&.selected.startRange,
-	&.startRange.startRange,
+	&.selected.startRange.endRange,
+	&.startRange.endRange,
+	&.selected.endRange.startRange,
 	&.endRange.startRange {
 		border-radius: 50px;
 
+		&:before {
+			border-radius: 50px;
```

Comment:
@dekkinga This is correct, but I would remove line 326 and instead do:

```
&,
&:before {
   border-radius: 50px;
}
```

It compiles the same thing, but it's less scss code 👍